golem_messages/datastructures/tasks.py

```diff
@@ -120,7 +120,7 @@ def sign_task(self, private_key: bytes) -> None:
     def verify(self, public_key: bytes) -> None:
         if self.signature is None:
             raise exceptions.InvalidSignature("No signature")
-        cryptography.ecdsa_verify(
+        return cryptography.ecdsa_verify(
```

Comment:
true, but on the other hand, it's natural to expect that value to be True in case the verification succeeds ... plus, I did that mostly for consistency with other `sign()` functions

Reply:
Diseases are also natural :) I would expect it to behave as declared in return type.